apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -9,7 +9,12 @@ import { __, sprintf } from '@wordpress/i18n';
 import { compose } from '@wordpress/compose';
 import { Button, Modal, Spinner, IconButton } from '@wordpress/components';
 import { registerPlugin } from '@wordpress/plugins';
-import { withDispatch, withSelect } from '@wordpress/data';
+import {
+	withDispatch,
+	withSelect,
+	select as wpSelect,
+	dispatch as wpDispatch,
```

Comment:
At a quick check, I don't see any conflicts with `select` and `dispatch`. Any reasons why we're renaming them?

Reply:
To make the linter happy.  LOL...  when I had them as just `select` and `dispatch` the linter was unhappy with the `select`s and `dispatch`s that were already in the connect/mapState functions in use.  So instead of committing with --no-verify or disabling the linter for those lines I figured I would just rename them.

Reply:
Oh wow, my search skills really screwed up there!
I have no idea why I didn't see the other `select`s and `dispatch`s... I'm kinda wondering if I was looking into a different file... 😅